prototype_source/quantization_in_pytorch_2_0_export_tutorial.rst

```diff
@@ -0,0 +1,375 @@
+and ``BackendConfig`` to specify the supported ways of quantization in their backend.
+This API covers most use cases relatively well, but the main problem is that this API is not fully extensible
+without involvement of the quantization team:
+
+-  This API has limitation around expressing quantization intentions for complicated operator patterns such as in the discussion of
+   `Issue-96288 <https://github.com/pytorch/pytorch/issues/96288>`__ to support ``conv add`` fusion.
+   Supporting ``conv add`` fusion also requires some changes to current already complicated pattern matching code such as in the
+   `PR-97122 <https://github.com/pytorch/pytorch/pull/97122>`__.
+-  This API also has limitation around supporting user's advanced quantization intention to quantize their model. For example, if backend
+   developer only wants to quantize inputs and outputs when the ``linear`` has a third input, it requires co-work from quantization
+   team and backend developer.
+-  This API uses ``QConfigMapping`` and ``BackendConfig`` as separate object. ``QConfigMapping`` describes user's
+   intention of how they want their model to be quantized. ``BackendConfig`` describes what kind of quantization a backend support.
+   ``BackendConfig`` is backend specific, but ``QConfigMapping`` is not. And user can provide a ``QConfigMapping``
+   that is incompatible with a specific ``BackendConfig``. This is not a great UX. Ideally, we can structure this better
+   by making both configuration (``QConfigMapping``) and quantization capability (``BackendConfig``) backend
+   specific. So there will be less confusion about incompatibilities.
+-  In ``QConfig``, we are exposing observer/fake_quant classes as an object for user to configure quantization.
+   This increases the things that user needs to care about, e.g. not only the ``dtype`` but also how the
+   observation should happen. These could potentially be hidden from user to make user interface simpler.
+
+To address these scalability issues, 
+`Quantizer <https://github.com/pytorch/pytorch/blob/3e988316b5976df560c51c998303f56a234a6a1f/torch/ao/quantization/_pt2e/quantizer/quantizer.py#L160>`__
+is introduced for quantization in PyTorch 2.0 export. ``Quantizer`` is a class that users can use to
+programmatically set the quantization specifications for input and output of each node in the model graph. It adds flexibility
+to the quantization API and allows modeling users and backend developers to configure quantization programmatically.
+This will allow users to express how they want an operator pattern to be observed in a more explicit
+way by annotating the appropriate nodes. A backend specific quantizer inherited from base quantizer,
+some methods that need to be implemented:
+
+-  `annotate method <https://github.com/pytorch/pytorch/blob/3e988316b5976df560c51c998303f56a234a6a1f/torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py#L269>`__
```

Comment:
is this the only method that needs to be implemented (above description says some methods)? Why create a separate bullet point for it in that case?

Reply:
Actually there are some other methods needs to be implemented. However, the detail design for the other methods are not fully set per the discussion with @jerryzh168. So we only mention `annotate method ` here. Sure, I will merge this bullet into previous paragraph for now.